### PR TITLE
added project quality tools + license

### PR DESCRIPTION
**Pull request by rvalyi**
added files: .coveragerc .gitignore LICENSE .travis.yml README.md

### CODE REVIEW

**Comment by rvalyi** [comment]
note: tests are failing because of flake8, what should I do, disable flake8 for now?
We will soon push a branch with flake8 passing on master, but as for 7.0 I'm afraid it won't be or top priority.
So what do the other fellow reviewers think?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml:16]: `${VERSION}` is not needed now with last maintainer version (OCB test included)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - sudo pip install python-ldap
+  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
+
+script:
+  - travis_run_flake8
+  - travis_run_tests ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml:20]: The same for `${VERSION}`

**Comment by pedrobaeza** [comment]
I think you should leave PEP8 tests as is. Somebody will fix it in the future, and it serves to check if an additional module contains PEP8 errors, reading between the rest of the errors meanwhile.

Regards.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
+  - sudo pip install python-ldap
+  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
```
Comment (bwrsandman) [inline .travis.yml]: Are these two lines necessary? They look like they are specific to server-tools

**Comment by bwrsandman** [comment]
:+1:

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
+  - $HOME/maintainer-quality-tools/travis/travis_install_nightly ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: This line is duplicated. Please remove it.

File: .travis.yml (hunk 1 above)
Comment (pedrobaeza) [inline .travis.yml:16]: Looking deep into the script, I see you still need the parameter, so don't take into account the comment.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**